initial_setup/gui/spokes/eula.py

```diff
@@ -0,0 +1,118 @@
+"""EULA spoke for the Initial Setup"""
+
+import gettext
+import logging
+
+from pyanaconda.ui.common import FirstbootOnlySpokeMixIn
+from pyanaconda.ui.gui.spokes import NormalSpoke
+from pyanaconda.core.constants import FIRSTBOOT_ENVIRON
+from pykickstart.constants import FIRSTBOOT_RECONFIG
+
+from initial_setup.product import eula_available, get_license_file_name
+from initial_setup.common import LicensingCategory
+
+log = logging.getLogger("initial-setup")
+
+_ = lambda x: gettext.ldgettext("initial-setup", x)
+N_ = lambda x: x
+
+__all__ = ["EULAspoke"]
+
+class EULAspoke(FirstbootOnlySpokeMixIn, NormalSpoke):
+    """The EULA spoke"""
+
+    builderObjects = ["eulaBuffer", "eulaWindow"]
+    mainWidgetName = "eulaWindow"
+    uiFile = "eula.glade"
+    # The EULA spoke is self-explanatory, so just redirect its
+    # help button to the Initial Setup Hub help file,
+    # which covers the overall theory of Initial Setup
+    # usage.
+    helpFile = "InitialSetupHub.xml"
+
+    icon = "application-certificate-symbolic"
+    title = N_("_LICENSE INFORMATION")
+    category = LicensingCategory
+    translationDomain = "initial-setup"
+
+    def initialize(self):
+        log.debug("initializing the EULA spoke")
+        NormalSpoke.initialize(self)
+
+        self._have_eula = True
+        self._eula_buffer = self.builder.get_object("eulaBuffer")
+        self._agree_check_button = self.builder.get_object("agreeCheckButton")
+        self._agree_label = self._agree_check_button.get_child()
+        self._agree_text = self._agree_label.get_text()
+
+        log.debug("looking for the license file")
+        license_file = get_license_file_name()
+        if not license_file:
+            log.error("no license found")
+            self._have_eula = False
+            self._eula_buffer.set_text(_("No license found. Please report this "
+                                         "at http://bugzilla.redhat.com"))
+            return
+
+        self._eula_buffer.set_text("")
+        itr = self._eula_buffer.get_iter_at_offset(0)
+        log.debug("opening the license file")
+        with open(license_file, "r") as fobj:
+            #fobj_lines = fobj.readlines()
```

Comment:
Please, remove the commented code in this method.

Reply:
Fixed, thanks! :)